frontend/package.json

```diff
@@ -5,18 +5,27 @@
   "scripts": {
     "dev": "next dev",
     "build": "next build && next export",
-    "lint": "next lint"
+    "lint": "next lint",
+    "test": "jest"
   },
   "dependencies": {
     "@next/font": "13.1.6",
     "@types/node": "18.13.0",
     "@types/react": "18.0.28",
     "@types/react-dom": "18.0.11",
+    "babel-jest": "^29.4.3",
     "eslint": "8.34.0",
     "eslint-config-next": "13.1.6",
+    "identity-obj-proxy": "^3.0.0",
```

Comment:
1. Why do we need this? Doesn't everything work without Babel using the Next.js Rust compiler (which is enabled by default since version 12, we have version 13 installed)
2. Shouldn't it be in `devDependencies`?